torchinductor/scheduler.py

```diff
@@ -183,6 +184,41 @@ def can_free(self):
                 return False
         return True
 
+    def codegen_originating_info(self, buffer, only_once=True):
+        if not config.comment_origin:
+            return
+
+        if only_once and self.written:
+            return
+        origins = self.node.origins
+        out_lines = []
+
+        for o in origins:
+            if o.op == "output":
+                # These are boring and samey
+                continue
+
+            out_lines.append("")
+            # TODO(voz): Should the pragma be constant somewhere?
+            out_lines.append("#pragma CMT ORIGIN:")
+            out_lines.append(f"#pragma CMT {o.op} {o.target}")
+            out_line_meta = f"{o.meta}"
```

Comment:
dumping the entire meta might  be too verbose... 
you can look at append_stacktrace_summary's impl in graph.py if you just want to show the summary...

Reply:
I like the whole meta, I think it's all useful. The whole thing is config disable-able anyway.